src/modules/postgres/setup/16-05-2016.update.sql

```diff
@@ -0,0 +1,46 @@
+UPDATE users SET identities = '{}' WHERE identities IS NULL;
```

Comment:
@harishkumarv I guess this will not work since there are no defaults set at this point? I tested on staging and it works fine if it's at the end.